pp_hot.c

```diff
@@ -1083,7 +1083,7 @@ PP(pp_multiconcat)
                     if (!SvOK(right)) {
                         if (ckWARN(WARN_UNINITIALIZED))
                             report_uninit(right);
-                        sv_setsv_flags(left, &PL_sv_no, 0);
+                        sv_setbool(left, FALSE);
```

Comment:
Here and in gv.h we go from:
```
sv_setsv_flags(left, &PL_sv_no, 0)
```
to:
```
sv_setsv_flags(left, &PL_SV_no, SV_GMAGIC|SV_DO_COW_SVSETSV)
```
since `sv_setbool(left, FALSE)` does `sv_setsv(sv, boolSV(b))`

... does this matter?

Reply:
It does not. `SV_GAMAGIC` tells `sv_setsv_flags()` to invoke GETMAGIC() on the source; but we know that `PL_sv_no` has no such magic, so this extra flag causes no problem.

In addition, `SV_DO_COW_SVSETSV` is the flag that tells `sv_setsv_flags()` to preserve the COW string and is in fact the exact flag we wanted here - the 0 wouldn't preserve it and hence would break the booleanness of it.